package.json

```diff
@@ -79,6 +80,7 @@
   "dependencies": {
     "auth0-lock": "^11.33.2",
     "ember-modal-dialog": "^4.0.1",
-    "moment": "^2.29.4"
+    "moment": "^2.29.4",
+    "date-fns": "^2.28.0"
```

Comment:
If you decided to using `date-fns` consider to remove `moment` and `ember-moment` from your `package.json` and migrate all `moment` things to `date-fn`